How to generate Google API keys.md

```diff
@@ -25,7 +25,7 @@ Here's how to get its 2 access keys: `clientId`, `refreshToken`
 
 	<img width="771" alt="Create credentials" src="https://user-images.githubusercontent.com/1402241/77865679-e89f3a00-722f-11ea-942d-5245091f22b8.png">
 
-0. Select **Chrome App** (or **Desktop app** if available), enter `chrome-webstore-upload` and click **Create** 
+0. Select **Chrome App** (**Desktop app** and **Other** also work, if Chrome App is missing), enter `chrome-webstore-upload` and click **Create** 
```

Comment:
So that means the `Other` type is still there?

Reply:
I think we had other people reporting they couldn't find one type or the other, so maybe Google shows different possible values to different people. I think it's ok to specify "Other" as an alternative.

Reply:
Yeah makes sense 👍